### DISCUSSION TITLE
Toggle for Sleek Menu Bar?

### DISCUSSION THREAD

**Discussion by Grydpype** [ransome1/sleek#630] [Q&A] [answered]
Dear all,
my apologies if this question has been raised before, but is there a toggle for the Sleek menu bar, or is there just no menu bar?

Regards,
kai.

**Comment by ransome1** [comment]
Edit: Sorry I didn't fully read the question. You are referring to the window menu, not `sleeks menu` (`navigation`), right?

If so; No, there is no toggle yet for the `window menu`.

**Reply by Grydpype** [reply]
The regular menu bar at the top of the sleek window; correct; which I saw in your OSX screenshot and which under linux I currently do not see. So if there's no toggle, that means there's no menu bar then, correct?

**Reply by ransome1** [reply, marked as answer]
The menu bar is there, but you need to reveal it pressing `alt key`. It is just not persisted. So on every start of sleek I assume it will be hidden again.

**Reply by Grydpype** [reply]
Thanks; will try that this evening.

Regards,
kai.
